task-runner/task_runner/executers/seissol/executer.py

```diff
@@ -0,0 +1,33 @@
+from task_runner.executers import ExecCommandLogger, MPIExecuter, mpi_configuration
+from task_runner.utils import loki
+
+
+class SeisSolExecuter(MPIExecuter):
```

Comment:
The class MPIExecuter is a bit overcomplicated and we plan on discontinuing it. Could you implement the executer with the class `BaseExecuter`? 

You can take a look into other examples and if you don't understand anything fell free to ask.

Reply:
okay will do that